lib/common_test/src/ct.erl

```diff
@@ -404,37 +404,37 @@ be specified as `Required`. Only the last element of the tuple can be a list of
 _Example 1._ Require the variable `myvar`:
 
 ```erlang
- ok = ct:require(myvar).
+ok = ct:require(myvar).
 ```
 
 In this case the configuration file must at least contain:
 
 ```text
- {myvar,Value}.
+{myvar,Value}.
 ```
```

Comment:
For example here you can see that it says 'text', but content of file is clearly an erlang term.

Reply:
Will be changed to erlang as well.